Competition2017/src/competition/subsystems/drive/commands/TankDriveWithJoysticksCommand.java

```diff
@@ -22,6 +22,13 @@ public void initialize() {
 
     @Override
     public void execute() {
-        driveSubsystem.tankDrivePowerMode(oi.leftJoystick.getVector().y, oi.rightJoystick.getVector().y);
+        double leftPower = oi.leftJoystick.getVector().y;
+        double rightPower = oi.rightJoystick.getVector().y;
+        
+        if(driveSubsystem.precisionMode){
```

Comment:
Might be better to do this in the drive subsystem's tankDrivePowerMode method so it benefits everyone without them having to do this too